demisto_sdk/commands/openapi_codegen/openapi_codegen.py

```diff
@@ -1009,7 +1008,7 @@ def clean_function_name(name: str, snakeify: bool = True) -> str:
 
         if snakeify:
             name = camel_to_snake(name)
-            name = name.replace('-', '_').replace('__', '_').strip('_')
+            name = name.replace('-', '_').replace('__', '_').replace('.', '').strip('_')

```

Comment:
I think`.` should be in `ILLEGAL_CODE_CHARS`

Reply:
It's wasn't there but I added it.